ts/screens/PinLoginScreen.tsx

```diff
@@ -117,6 +120,10 @@ class PinLoginScreen extends React.Component<Props> {
     };
     return (
       <BaseScreenComponent primary={true} contextualHelp={contextualHelp}>
+        <StatusBar
```

Comment:
this adds the status bar to all screens based on `BaseScreenComponent` (almost all), is that what we want?

Reply:
@cloudify He is not adding something to the BaseScreenComponent file, we are in the PinLoginScreen. Also the StatusBar component is not really a "rendered" component, you can put 10 StatusBar in each screen and only one is rendered, the last one dictate the android/ios bar background/text color.

Reply:
sorry you are right :)